src/Dialogue/Confirm.php

```diff
@@ -8,6 +8,137 @@
 class Confirm extends Dialogue
 {
 
+    private $yesText = 'Yes';
```

Comment:
I think instead of changing the `Confirm` behaviour we could have a new dialogue type? maybe a `YesNo` dialogue ?